velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -372,13 +385,29 @@ std::vector<MaterializedRow> materialize(
   std::vector<MaterializedRow> rows;
   rows.reserve(size);
 
+  // Pre-compute null decimal values for all columns.
```

Comment:
null values can be created for all column types not just decimal here and can be reused. Lets change this.

Reply:
Updated, thanks for the suggestion.